### DISCUSSION TITLE
7-Eleven stores were temporarily converted into Kwik E-marts to promote the release of what movie?

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#428] [Galaxy Trivia] [answered]
**Bonus:** What year did this happen?

**Comment by peterbaccaro** [comment]
7-Eleven stores were temporarily converted into Kwik E-marts to promote the release of **The Simpsons Movie**.
The 7-Eleven stores were temporarily converted into Kwik E-marts in **July 2007**.

**Comment by DrakonZZZ** [comment]
July 2007

**Comment by thuongtruong1009** [comment, marked as answer]
In 2007, 7-Eleven stores in the United States were temporarily converted into Kwik-E-Marts as part of a promotion for the release of the movie "The Simpsons Movie". The Kwik-E-Mart is a fictional convenience store that appears in the animated television show "The Simpsons", and the promotion allowed fans to experience a real-life version of the store. The promotion included products inspired by the show, such as Squishees (a parody of Slurpees) and Krusty-O's cereal. The Kwik-E-Mart promotion was considered a successful marketing campaign and helped to generate excitement for the release of "The Simpsons Movie".
